### PR TITLE
Add JS test command to build steps

### PR DESCRIPTION
**Pull request by steverydz**
## Done

Add JS testing as a build step

### CODE REVIEW

**Comment by steverydz** [comment]
@nottrobin now we have a (very) modest JS test suite it makes sense to add it as a step in the build process. Based on the other entries in the CircleCI config I'm sure this is fine but I don't know how to test it.

**Comment by nottrobin** [comment]
Nice! Where's the actual test code? I'm struggling to find it.

**Comment by steverydz** [comment]
> Nice! Where's the actual test code? I'm struggling to find it.

There's only two at the moment. Anything with `.test.js` at the end basically:

https://github.com/canonical-web-and-design/ubuntu.com/tree/master/static/js/src

File: .circleci/config.yml [hunk 1]
```diff
@@ -86,6 +86,16 @@ jobs:
...
+      - checkout
+      - run:
+          name: Install dependencies
+          command: yarn
+      - run:
+          name: Test JS
+          command: yarn test-js
 workflows:
   version: 2
   build:
```
Comment (nottrobin) [inline .circleci/config.yml]: To get this running in CI you need to add `- test-js` to the `jobs:` list below.

Comment (steverydz) [reply .circleci/config.yml]: done

**Pull request merged**